.circleci/config.yml

```diff
@@ -40,3 +43,37 @@ jobs:
           | python3 - --version 1.1.0"
       - run: poetry build
       - run: POETRY_HTTP_BASIC_PYPI_PASSWORD=${PYPI_PASSWORD} poetry publish
+  test_installation:
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install python3 python3-dev curl git -y
+      - checkout
+      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
+          | python3 - --version 1.1.0"
+      - run: echo "export PATH=$PATH:$HOME/.poetry/bin" >> $BASH_ENV
+      - run: sudo $(which poetry) install
+      - run: sudo $(which poetry) run brainframe install --noninteractive
+      - run: sudo $(which poetry) run brainframe compose up -d
+      - run:
+          name: Check BrainFrame containers are running
+          command: |
+            for i in `seq 1 5`; do
+              if docker container inspect -f {{.State.Running}}  $(docker ps -q -f name="brainframe_core"); then
+                exit 0;
+              fi
+              sleep 1
+            done
+            exit 1
```

Comment:
I think it would be nice to print an explanatory message before calling `exit 1`. Maybe something like `echo "The core service failed to start"`

Reply:
What do you think  about using this pattern instead? (Pulled from [this stackoverflow](https://stackoverflow.com/questions/21183088/how-can-i-wait-for-a-docker-container-to-be-up-and-running))

```
until [ "`docker inspect -f {{.State.Running}} CONTAINERNAME`"=="true" ]; do
    sleep 0.1;
done;
```

Then you can limit the circleci step timeout manually to be 1 minute  
```
- run:  
    command: 
    no_output_timeout: 1m
```

My suggestion is just because I'd prefer less bash for loops when possible ;)

Reply:
@apockill "no_output_timeout" only works when there is no output. So I have redirected the outputs from shell commands, it should work now. 

@velovix Probably I can't do it since I'm using CircleCI's timeout mechanism now. It will give you something like "Too long with no output (exceeded 1m0s): context deadline exceeded"

Reply:
That's fine by me